### ISSUE TITLE
Unable to create fixed size array with 1 element and initializer

### ISSUE THREAD

**Issue by damianday** [beefytech/Beef#65]
```
int[1] indexArr = ( 1 ); // Fails
int[2] indexArr2 = ( 1, 2 ); //Succeeds
```

**Comment by bfiete** [comment]
This was a "secondary" initialization style which technically is initializing from tuples, and "(1)" is not a tuple expression, it's a parenthazied expression.

The primary initialization style for sized arrays is to use the inferred-constructor style, and that also works for assignments:

int[1] indexArr = .(1);
int[2] indexArr2 = .(1, 2);
indexArr = .(3);
indexArr2 = .(3, 4);

Technically these are shorthands for the explicit sized array constructor style as such:
int[2] indexArr2 = int[2](1, 2);

Anyway. The tuple initialization style predated the "." inferred syntax, and I have removed it now since it doesn't really make sense to initialize from tuples and the proper syntax is only one character longer.  Removed in https://github.com/beefytech/Beef/commit/5bed292e87201a33869f5cf7453cad50c557b644
